frontend/src/components/ItemNavigation.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {ChangeEvent, useState} from 'react';
+import axios from "axios";
+import "../css/ItemNavigation.css"
+
+type itemNavigationProbs = {
+    fetchData: () => void
+    setErrorModal: (showErrorAlert: boolean) => void
+    setErrorMessage: (errorMessage: string) => void
+    setSuccessMessage: (successMessage: string) => void
+    setSuccessModal: (showSuccessAlert: boolean) => void
+}
+
+function ItemNavigation(props: itemNavigationProbs) {
+
+    const [ean, setEan] = useState<string>("")
+    const [itemNumber, setItemNumber] = useState<string>("")
+    const [storableValue, setStorableValue] = useState<string>("")
```

Comment:
Wenn der Typ des useState dem Typ des explizit angegebenen Initialwerts entspricht, dann kann TypeScript den Typen selbst ermitteln.
Ist aber Geschmackssache, ob man ihn trotzdem angibt, oder nicht.
```suggestion
    const [ean, setEan] = useState("")
    const [itemNumber, setItemNumber] = useState("")
    const [storableValue, setStorableValue] = useState("")
```